src/setupTests.js

```diff
@@ -8,3 +8,10 @@ global.React = React;
 global.shallow = shallow;
 global.render = render;
 global.mount = mount;
+global.MutationObserver = class {
```

Comment:
What these lines of code do? Just try to explain

Reply:
While I was testing, I realized that React quill needs mutation observer in its testing. Mutation observer helps in tracking changes while typing.

You can refer to this issue  https://github.com/zenoamaro/react-quill/issues/434